docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,270 @@
+=================
+Pulsar connector
+=================
+`Apache Pulsar <https://pulsar.apache.org/en/>`_ is a cloud-native, distributed messaging and streaming platform.
+
+This connector allows the use of Pulsar topics as tables in Trino, such that each Pulsar message
+in a Pulsar topic is presented as a row in Trino. And Pulsar topic schema is mapped to Trino
+table schema.
+
+Topics in Pulsar are stored as segments in `Apache BookKeeper <https://bookkeeper.apache.org/>`_. Each topic segment is replicated to some BookKeeper nodes,
+which enables concurrent reads and high read throughput. You can configure the number of BookKeeper nodes,
+and the default number is 3. In Trino Pulsar connector, data is read directly from BookKeeper,
+so Trino workers can read concurrently from horizontally scalable number BookKeeper nodes.
+
+This connector is **read-only** for now. It can only read data from Pulsar topics,
+but cannot create topic or write to exiting topic yet.
+
+To configure the Pulsar connector, create a catalog properties file ``etc/catalog/pulsar.properties``
+A sample is provided:
+
+.. code-block:: text
+
+    connector.name=pulsar
+    pulsar.broker-service-url=http://localhost:8080
+    pulsar.zookeeper-uri=127.0.0.1:2181
+    pulsar.max-entry-read-batch-size=100
+    pulsar.target-num-splits=2
+    pulsar.max-split-message-queue-size=10000
+    pulsar.max-split-entry-queue-size=1000
+    pulsar.max-split-queue-cache-size=-1
+    pulsar.namespace-delimiter-rewrite-enable=false
+    pulsar.rewrite-namespace-delimiter=/
+
+    pulsar.managed-ledger-offload-driver = aws-s3
+    pulsar.offloaders-directory = /pulsar/offloaders
+    pulsar.managed-ledger-offload-max-threads = 2
+
+    pulsar.offloader-properties = \
+      {"s3ManagedLedgerOffloadBucket": "offload-bucket", \
+      "s3ManagedLedgerOffloadRegion": "us-west-2", \
+      "s3ManagedLedgerOffloadServiceEndpoint": "http://s3.amazonaws.com"}
+
+    pulsar.auth-plugin =
+    pulsar.auth-params =
+    pulsar.tls-allow-insecure-connection =
+    pulsar.tls-hostname-verification-enable =
+
+    pulsar.tls-trust-cert-file-path =
+
+    pulsar.bookkeeper-throttle-value = 0
+    pulsar.bookkeeper-num-io-threads =
+    pulsar.bookkeeper-num-worker-threads =
+    pulsar.bookkeeper-use-v2-protocol=true
+    pulsar.bookkeeper-explicit-interval=0
+
+    pulsar.managed-ledger-cache-size-MB = 0
+    pulsar.managed-ledger-num-worker-threads =
+    pulsar.managed-ledger-num-scheduler-threads =
+
+    pulsar.stats-provider-configs={"httpServerEnabled":"false", "prometheusStatsHttpPort":"9092", "prometheusStatsHttpEnable":"true"}
+
+The following configuration properties are available:
+
+================================================      =======================================================================
+Property Name                                         Description
```

Comment:
Please mention default values (as separate column)?

I guess this format for table is obsolete and we use something else. @mosabua any ideas?

Reply:
will update.